src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2,6 +2,6 @@ databaseChangeLog:
   - include:
       file: /db/changelog/create/0001_address.sql
   - include:
-      file: /db/changelog/create/0002_approved_postal_application.sql
+      file: /db/changelog/create/0002_approved_postal_vote_application.sql
   - include:
-      file: /db/changelog/create/0003_approved_proxy_application.sql
+      file: /db/changelog/create/0003_approved_proxy_vote_application.sql
```

Comment:
Do we maybe want to just call these xxxx_applications - we know we'll be sending both approved and rejected and could a more painful change later.